bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/seqcomp/GhostSeqRepeatFlatObsCB.java

```diff
@@ -0,0 +1,110 @@
+package edu.gemini.spModel.gemini.seqcomp;
+
+import edu.gemini.pot.sp.ISPSeqComponent;
+import edu.gemini.spModel.config.AbstractSeqComponentCB;
+import edu.gemini.spModel.data.config.*;
+import edu.gemini.spModel.dataflow.GsaSequenceEditor;
+import edu.gemini.spModel.gemini.calunit.CalUnitConstants;
+import edu.gemini.spModel.gemini.calunit.CalUnitParams.*;
+import edu.gemini.spModel.gemini.calunit.calibration.CalConfigBuilderUtil;
+import edu.gemini.spModel.gemini.ghost.Ghost$;
+import edu.gemini.spModel.obscomp.InstConstants;
+import edu.gemini.spModel.seqcomp.GhostSeqRepeatExp;
+import edu.gemini.spModel.seqcomp.SeqConfigNames;
+import edu.gemini.spModel.seqcomp.SeqRepeatCbOptions;
+
+import java.util.Map;
+
+/**
+ * A configuration builder for the Gemini CalUnit sequence component for GHOST.
+ */
+public class GhostSeqRepeatFlatObsCB extends AbstractSeqComponentCB {
```

Comment:
`final`?

Reply:
Gah. Yes. I'll go through and finalize it all in another PR.